src/utils.ts

```diff
@@ -67,35 +82,63 @@ function printElement({ props }) {
   )}`;
 }
 
-function display(value) {
+function display(value: any) {
   return typeof value === 'string' ? value : stringify(value);
 }
 
-function getMessage(matcher, expectedLabel, expectedValue, receivedLabel, receivedValue) {
+function getMessage(
+  matcher: string,
+  expectedLabel: string,
+  expectedValue: string | RegExp,
+  receivedLabel: string,
+  receivedValue: string | null,
+) {
   return [
     `${matcher}\n`,
     `${expectedLabel}:\n${expectedColor(redent(display(expectedValue), 2))}`,
     `${receivedLabel}:\n${receivedColor(redent(display(receivedValue), 2))}`,
   ].join('\n');
 }
 
-function matches(textToMatch, matcher) {
+function matches(textToMatch: string, matcher: string | RegExp) {
   if (matcher instanceof RegExp) {
     return matcher.test(textToMatch);
-  } else {
-    return textToMatch.includes(String(matcher));
   }
+
+  return textToMatch.includes(matcher);
 }
 
-function normalize(text) {
+function normalize(text: string) {
   return text.replace(/\s+/g, ' ').trim();
 }
 
-function mergeAll(objects) {
-  return Object.assign({}, ...(objects ?? [{}]));
+type StyleRecord = {
+  [P in keyof TextStyle | keyof ViewStyle]?: TextStyle[P];
+};
+
+function getStylePropAsRecord(styles: Falsy | Object | Object[]): StyleRecord {
+  if (!styles) return {};
+
+  if (Array.isArray(styles)) {
+    const flatStyles = styles.flat();
+    let result = {};
+
+    for (const styleItem of flatStyles) {
+      const entries = Object.entries(styleItem);
+
+      for (const [key, value] of entries) {
+        // @ts-ignore how to type it?
+        result[key] = value;
+      }
+    }
+
+    return result;
+  }
+
+  return styles as StyleRecord;
 }
 
-function isEmpty(value) {
+function isEmpty(value: any) {
```

Comment:
```suggestion
function isEmpty(value: unknown) {
```

Reply:
Fixed, added eslint rule for the future that checks explicit any.